app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+    validates :first_name, presence: true, length:{maximum: 30},format: { with: /\A[a-zA-Z]+\z/}
+    validates :last_name, presence: true, length:{maximum: 30}, format: { with: /\A[a-zA-Z]+\z/}
+    validates :phone_number, presence: true, uniqueness: true, length: {in: 10..13},format: { with: /\A[a-zA-Z]+\z/}
```

Comment:
Phone number needs to be a number not letters 😄 so the regex needs to be `/\d+/` 

`\d` means any digit
`+` means more than one

Reply:
noted will redo it